### PR TITLE
chore(statics): onboard new batch of tokens

### PR DESCRIPTION
**Pull request by grsind19**
Ticket: BG-70350

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -7443,6 +7451,18 @@ export const coins = CoinMap.fromCoins([
     undefined,
     Networks.test.goerli
   ),
+  terc20(
+    'c28caef6-1312-49ad-9aad-fff8fdb63c6c',
+    'theu',
+    'Goerli Test Himalaya Euro Token 18 Decimals',
```
Comment (aarvay) [inline modules/statics/src/coins.ts:7457]:
```suggestion
    'Goerli Test Himalaya Euro Token',
```

**Review by aarvay — CHANGES_REQUESTED** [review]

**Review by sachushaji — APPROVED** [review]

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -1465,6 +1465,14 @@ export const coins = CoinMap.fromCoins([
     '0x1a4b46696b2bb4794eb3d4c26f1c55f9170fa4c5',
     UnderlyingAsset.BIT
   ),
+  erc20(
+    '83214bf2-eec7-4225-b398-41e065e23311',
+    'blur',
+    'Blur',
+    18,
+    '0x5283D291DBCF85356A21bA090E6db59121208b44',
```
Comment (sachushaji) [inline modules/statics/src/coins.ts:1473]: All contract addresses have to be in lowerCase

**Review by sachushaji — CHANGES_REQUESTED** [review]
Hey @grsind19 , Thanks for the PR. All contract addresses needs to be lowercased, other wise things looks okay to me

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (grsind19) [reply modules/statics/src/coins.ts]: @aarvay updated the changes

File: modules/statics/src/coins.ts (hunk 2 above)
Comment (grsind19) [reply modules/statics/src/coins.ts]: @sachushaji updated the contract address to lower case

**Comment by grsind19** [comment]
@sachushaji completed the review changes, please verify

**Review by sachushaji — APPROVED** [review]
LGTM, Thanks for addressing the changes @grsind19

**Comment by grsind19** [comment]
@aarvay  could you please review the changes

**Pull request merged**